content/news/2023-12-22-esg-wp3-update/index.md

```diff
@@ -0,0 +1,105 @@
+
+<div class="center">
+<div class="img-sizer" style="width: 40%">
+
+![The ESG Pulsar Network contributors](esg_pulsar_network_contrib.png)
+
+</div>
+<figcaption>
+  EuroScienceGateway Work Package 3 pulsar endpoints contributors.
+</figcaption>
+</div>
+
+<br/>
+<br/>
+
+First of all we updated the components of the Open Infrastructure to the latest available software revision. The Open Infrastructure is a set of tools allowing to provide ready-to-go Pulsar nodes and Galaxy servers deployable over partners Cloud infrastructures.
+
+Currently, each Pulsar endpoint is created using a virtual machine image, named Virtual Galaxy Compute Nodes, which provides everything needed. Containerized tools and Reference Data are shared through read only repositories based on the CERN-VM FileSystem. Finally, the deployment and the configuration of the Pulsar endpoint is managed using Terraform and Ansible. HTCondor is used as the default Resource Manager for job submission.
+
+<div class="center">
+<div class="img-sizer" style="width: 40%">
+
+![Pulsar endpoint architecture](pulsar_endpoint_arch.png)
+
+</div>
+<figcaption>
+  Pulsar endpoint architecture.
+</figcaption>
+</div>
+
+<br/>
+<br/>
+
+The Open infrastructure update includes:
+* Images updated to RockyLinux 9 ([repository](https://usegalaxy.eu/static/vgcn/))
+* Pulsar app updates.
+* HTCondor updated to 10.4.3 (with the new authentication mechanism between compute nodes and the Central Manager). 
+* [Continuous configuration automation](https://github.com/usegalaxy-eu/pulsar-deployment)
+* Documentation update
+These activities resulted in the possibility to deploy new Pulsar endpoints with just a single  command, with the only requirements being access to an  OpenStack tenant and the UseGalaxy.eu RabbitMQ credentials.
+
+Most Pulsar endpoints have been already updated to the new configuration:
+* ALU-FR (DE) Pulsar deployments for testing
+* CNR (IT) - IT02 and IT03
+* IISAS (Slovakia) - SK01
+* CNRS - GenOuest (FR) - FR01 
+* CESNET (Czech Republic) - CZ01
+* EGI and INFN - EGI01
+* BSC (ES) - ES01
+* TUBITAK ULAKBIM setup is almost completed
+
+Moreover, the updated Open Infrastructure allows to deploy full fledged usegalaxy.eu replica servers, thus allowing to instantiate new usegalaxy services easily, but also providing a robust framework for maintaining and updating running instances. We also started documenting the whole procedure, which has been successfully used to deploy a prototype of the Italian UseGalaxy server ([temporary repository](https://usegalaxy-it.github.io/documentation/)).
+
+Currently, 7 UseGalaxy endpoints are available: EU (ALU-FR), Be (VIB), Fr (CNRS), Es (BSC-CNS), No (ELIXIR-NO), Cz (CESNET) and It (CNR). In this case the activities focused on the management and maintenance of the instances:
+* Align to Galaxy 23.0+ release
```

Comment:
Is align = Update to 23.0 release? Are we not already at 23.1 or is that meant by 23.0+?

Reply:
Currently, only EU is at 23.1. The others have updated to 23.0 recently.